types/lodash/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Lo-Dash 4.14
+// Type definitions for Lo-Dash 4.15
```

Comment:
I would leave the version number at 4.14. We don't follow semver in DT, instead we try to keep the version number (at least the major.minor) in sync with the JS package. It's pretty much impossible to follow semver for typings since nearly every change is a breaking change (as you probably noticed).